File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/typescript/ADV-18.ts
declare function require(m: string): any;

function pick(b: boolean): bigint {
    if (b) {
        return 1n;
    }
}

console.log("ADV-START");

const lines: string[] = require("fs").readFileSync(0, "utf8").split("\n");
const b: boolean = lines[0] === "1";

const r: bigint = pick(b) * 2n;

console.log("OBS=R:" + String(r));
console.log("ADV-END");

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/go/ADV-18.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func pick(b bool) int64 {
	if b {
		return 1
	}
}

func main() {
	fmt.Println("ADV-START")

	sc := bufio.NewScanner(os.Stdin)
	sc.Scan()
	b := sc.Text() == "1"

	r := pick(b) * 2

	fmt.Printf("OBS=R:%d\n", r)
	fmt.Println("ADV-END")
}
